OpenKeychain/src/main/res/values/arrays.xml

```diff
@@ -38,8 +38,8 @@
     </string-array>
     <string-array name="theme_values" translatable="false">
         <item>"auto"</item>
-        <item>"dark"</item>
         <item>"light"</item>
+        <item>"dark"</item>
```

Comment:
I understand this was the least changes needed to correct the reversed themes, but the ordering bothers me.

Is there a style guide that prefers dark after light instead of using alphabetical ordering?